UndertaleModTool/Scripts/UTDR Scripts/EnableDeltaruneDebugMod.csx

```diff
@@ -0,0 +1,165 @@
+EnsureDataLoaded();
+
+ScriptMessage("Enabling debug mode");
+
+UndertaleModLib.Compiler.CodeImportGroup importGroup = new(Data);
+
+string displayName = Data.GeneralInfo.DisplayName.Content;
+
+if (displayName == "DELTARUNE Chapter 1")
+{
+    ScriptMessage("Detected DELTARUNE Chapter 1 - modifying obj_initializer2");
+    
+    var obj_initializer2 = Data.GameObjects.ByName("obj_initializer2");
+    if (obj_initializer2 == null)
+    {
+        ScriptError("Could not find obj_initializer2");
+        return;
+    }
+
+    var createCode = obj_initializer2.EventHandlerFor(EventType.Create, (uint)0, Data);
+    if (createCode == null)
+    {
+        ScriptError("Could not find Create event for obj_initializer2");
+        return;
+    }
+
+    importGroup.QueueFindReplace(createCode,
+        "global.debug = 0;",
+        "global.debug = 1;"
+    );
+
+    importGroup.Import();
+    ChangeSelection(createCode);
+    
+    ScriptMessage("Debug mode is now permanently enabled for DELTARUNE Chapter 1! Coded By Cyn-ically");
+}
+else if (displayName == "DELTARUNE Chapter 2")
+{
+    ScriptMessage("Detected DELTARUNE Chapter 2 - modifying multiple objects");
```

Comment:
I think this may also need `cutsceneshow = false;` added to `gml_Object_obj_debugProfiler_Create_0`, if it's an empty event, as occurs in 1.09+ versions of the chapter, in order to prevent crashes. (See the `Ch2 Debug.csx` script that currently exists.)